packages/rum/src/domain/record/record.spec.ts

```diff
@@ -422,6 +422,18 @@ describe('record', () => {
     })
   })
 
+  describe('it should not record when full snapshot is pending', () => {
+    it('ignores any record while a full snapshot is pending', () => {
+      mockRequestIdleCallback()
+      startRecording()
+      newView()
+
+      window.dispatchEvent(createNewEvent('focus'))
+
+      expect(emitSpy).toHaveBeenCalled()
```

Comment:
**💬 suggestion:** ‏it could be more explicit like this:
```ts
      mockExperimentalFeatures([ExperimentalFeature.ASYNC_FULL_SNAPSHOT])
      mockRequestIdleCallback()
      startRecording()
      newView()

      emitSpy.calls.reset()

      window.dispatchEvent(createNewEvent('focus'))

      expect(getEmittedRecords().find((record) => record.type === RecordType.Focus)).toBeUndefined()
```